application/src/main/java/com/musicforall/model/Track.java

```diff
@@ -147,6 +147,10 @@ public void setLocation(String location) {
         this.location = location;
     }
 
+    public String getEntireName() {
+        return getArtist() + " - " + getTitle();
```

Comment:
what about cases when artist or title is null?
also that kind of logic should not be present in domain objects